SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -42,12 +42,32 @@ FLocalDeploymentManager::FLocalDeploymentManager()
 
 		// Watch the worker config directory for changes.
 		StartUpWorkerConfigDirectoryWatcher();
+	}
+#endif // PLATFORM_WINDOWS
+}
 
-		// Restart the spatial service so it is guaranteed to be running in the current project.
-		AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this]
+void FLocalDeploymentManager::Init(FString RuntimeIPToExpose)
+{
+#if PLATFORM_WINDOWS
```

Comment:
Realise you didn't change this, but please add the // PLATFORM_WINDOWS comment to the closing #endif